adding aardvark benchmarking (#315)
nf-core/variantbenchmarking: CHANGELOG.md

### `Added`

- Template update for nf-core/tools v4.0.2 + a few fixes around [#305](https://github.com/nf-core/variantbenchmarking/issues/305)
- Aardvark Benchmarking Subworkflow: Introduced the AARDVARK_BENCHMARK subworkflow for sequence-resolved small variant (SNV/Indel) benchmarking [#315](https://github.com/nf-core/variantbenchmarking/issues/315).

> - Replaced Small Benchmark with somatic somall and germline small benchmark subworkflows: Added Aardvark as a evaluation method within the SMALL_BENCHMARK workflow, alongside Hap.py, Sompy, and RTGtools for both germline and somatic tracks [#315](https://github.com/nf-core/variantbenchmarking/issues/315).

Reviewer: ```suggestion.
- Replaced  SMALL_GERMLINE_BENCHMARK and SMALL_SOMATIC_BENCHMARK subworkflows with SMALL_BENCHMARK to include Aardvark  alongside Hap.py, Som.py, and RTGtools vcfeval for both germline and somatic tracks [#315](https://github.com/nf-core/variantbenchmarking/issues/315).
```
